mteb/tasks/Reranking/multilingual/WikipediaRerankingMultilingual.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+_EVAL_LANGS = {
+    "bg": ["bul-Cyrl"],
+    "bn": ["ben-Beng"],
+    "cs": ["ces-Latn"],
+    "da": ["dan-Latn"],
+    "de": ["deu-Latn"],
+    "en": ["eng-Latn"],
+    "fa": ["fas-Arab"],
+    "fi": ["fin-Latn"],
+    "hi": ["hin-Deva"],
+    "it": ["ita-Latn"],
+    "nl": ["nld-Latn"],
+    "pt": ["por-Latn"],
+    "ro": ["ron-Latn"],
+    "sr": ["srp-Cyrl"],
+}
+_CONFIG_LIST = list(_EVAL_LANGS.keys())
+
+class WikipediaRerankingMultilingual(AbsTaskReranking):
+    metadata = TaskMetadata(
+        name="WikipediaRerankingMultilingual",
+        description="The dataset is derived from Cohere's wikipedia-2023-11 dataset and contains synthetically generated queries.",
+        reference="https://huggingface.co/datasets/ellamind/wikipedia-2023-11-reranking-multilingual",
+        hf_hub_name="ellamind/wikipedia-2023-11-reranking-multilingual",
+        dataset={
+            "path": "ellamind/wikipedia-2023-11-reranking-multilingual",
+            "revision": "b20adcd11a02dc0226edbe8b5951e99ee0bb091c",
+            # "config": _CONFIG_LIST,
+        },
+        type="Reranking",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=_EVAL_LANGS,
+        main_score="map",
+        date=("2023-11-01", "2024-05-15"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=[],
+        license="cc-by-sa-3.0",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="LM-generated and verified",
```

Comment:
```suggestion
        annotations_creators="derived", # is this LM-generated as well. I would probably argue that it is.
        dialect=[],
        text_creation="LM-generated and verified",
```

Reply:
there only these options for annotator type:
```python
ANNOTATOR_TYPE = Literal["expert-annotated", "human-annotated", "derived"]
```

Reply:
You can add new ones to the list

Reply:
added a new "LM-generated" annotator type